pycc/tests/test_036_lr.py

```diff
@@ -0,0 +1,78 @@
+"""
+Test CCSD linear response functions.
+"""
+
+# Import package, test suite, and other packages as needed
+import psi4
+import numpy as np
+import pytest
+import pycc
+from ..data.molecules import *
+
+def test_linresp():
+    psi4.set_memory('2 GiB')
+    psi4.core.set_output_file('output.dat', False)
+    psi4.set_options({'basis': 'aug-cc-pvdz',
+                      'scf_type': 'pk',
+                      'e_convergence': 1e-12,
+                      'd_convergence': 1e-12,
+                      'r_convergence': 1e-12
+    })
+    mol = psi4.geometry(moldict["H2O"])
+    rhf_e, rhf_wfn = psi4.energy('SCF', return_wfn=True)
+
+    e_conv = 1e-12
+    r_conv = 1e-12
+
+    cc = pycc.ccwfn(rhf_wfn)
+    ecc = cc.solve_cc(e_conv, r_conv)
+    hbar = pycc.cchbar(cc)
+    cclambda = pycc.cclambda(cc, hbar)
+    lecc = cclambda.solve_lambda(e_conv, r_conv)
+    density = pycc.ccdensity(cc, cclambda)
+
+    resp = pycc.ccresponse(density)
+
+    omega1 = 0.0656
+
+    # Creating dictionaries
```

Comment:
I don't want the user interface to be so complicated to obtain the polarizability, but we can stick with this for now and make those changes on another PR.

Reply:
Okay